src/frontend/org/voltdb/calciteadapter/rules/physical/VoltDBPSortIndexScanRemoveRule.java

```diff
@@ -55,7 +55,14 @@ public void onMatch(RelOptRuleCall call) {
             AccessPath accessPath = scan.getAccessPath();
             accessPath.setSortDirection(sortDirection);
 
-            VoltDBPTableIndexScan newScan = VoltDBPTableIndexScan.create(
+//            Table catTableable = scan.getVoltDBTable().getCatTable();
```

Comment:
I guess you can remove this commented out code?

Reply:
Removed